contracts/splits/src/contract.rs

```diff
@@ -60,16 +63,15 @@ pub fn execute_distribute(
 ) -> Result<Response, ContractError> {
     let config = CONFIG.load(deps.storage)?;
 
-    // only a member can distribute funds
-    let weight = config
-        .group_addr
-        .is_member(&deps.querier, &info.sender, None)?
-        .ok_or(ContractError::Unauthorized {})?;
-    if weight == 0 {
-        return Err(ContractError::InvalidWeight { weight });
-    }
+    config.authorize(&deps.querier, &info.sender)?;
```

Comment:
I think would be nice to allow to be either an executor or any of the members, but seems like with this change you have to stick with one method, I guess can always set a bot to have 0 weight.

Reply:
Yes a 0 weight member makes more sense for the bot imho.